docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -2,13 +2,13 @@
 = Migrating components
 
 [role="_abstract"]
-This section describes how to migrate components to devfile v2. There ARE major changes in the `components` section of the devfile. A component specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
+This section describes how to migrate components to devfiles v2.x. A component specified in a devfile v1.x no longer works in a devfile v2.x
 
 .Procedure
 
 . Component is a polymorphic type
 +
-For a better syntax validation component now is defined as a polymorphic type and can be implemented as `container`, `kubernetes`, `openshift`, `plugin` or `volume`.
+For a better syntax, the validation component is now defined as a polymorphic type and can be implemented as `container`, `kubernetes`, `openshift`, `plugin` or `volume`.
```

Comment:
Not sure what "for a better syntax" adds to this (and the article feels unnecessary in any case).

Reply:
Eliminated "for a better syntax" I kept "_the_ validation component..." do you think it reads fine?

Reply:
Reads well to me. :)

Reply:
resolving this conversation then!